packages/firebase/index.d.ts

```diff
@@ -5853,6 +5853,19 @@ declare namespace firebase.firestore {
      * CACHE_SIZE_UNLIMITED to disable garbage collection.
      */
     cacheSizeBytes?: number;
+
+    /**
+     * Forces the SDK’s underlying network transport (WebChannel) to use
+     * long-polling. Each response from the backend will be closed immediately
+     * after the backend sends data (by default responses are kept open in
+     * case the backend has more data to send). This avoids incompatibility
+     * issues with certain proxies, antivirus software, etc. that incorrectly
+     * buffer traffic indefinitely. Use of this option will cause some
+     * performance degradation though.
+     *
+     * This setting may be removed in a future release.
```

Comment:
Should we invite users to discuss with us when they're using this?

The backend will have logs of when this is actually used, but if the rest of the world starts using this by default we won't know why otherwise.